src/devices/Buzzer/README.md

```diff
@@ -23,7 +23,7 @@ Here's an example how you could use `Buzzer`.
 ```csharp
 using (Buzzer buzzer = new Buzzer(21)); // Initialize buzzer with software PWM connected to pin 21.
 {
-	buzzer.PlayTone(440, 1000); // Play tone with frequency 440 hertz for one second.
+ buzzer.PlayTone(440, 1000); // Play tone with frequency 440 hertz for one second.
```

Comment:
same question about the indent

Reply:
catching up all of them now to replace the old tab which has been replaced by simple space to be aligned with the rest with 4 spaces